docs/orchestration/concepts/api.md

```diff
@@ -150,10 +142,10 @@ client.graphql(
 )
 ```
 
-To use `TENANT`-scoped tokens in programmatic requests, include the token as the authorization header of your GraphQL requests:
+To use `TENANT`-scoped keys in programmatic requests, include the key as the authorization header of your GraphQL requests:
```

Comment:
this line is still not 100% correct, since it references tenant-scoped. Should it be user or service account?